packages/web-reporter/writeReport.test.ts

```diff
@@ -0,0 +1,70 @@
+import { Measure, TestCaseIterationResult } from "@perf-profiler/types";
+import { getMeasuresForTimeInterval, writeReport } from "./writeReport";
+
+const mockMeasure = (name: string) => {
+  // We're just mocking measure to make tests more readable here
+  return name as unknown as Measure;
+};
+
+const mockResultIteration = (name: string[]) => ({
+  measures: name.map(mockMeasure),
+  time: 0,
+});
+
+test("getMeasuresForTimeInterval", () => {
```

Comment:
Maybe add tests:
  - if duration is null 
  - if skip or duration is not a multiple of 500